src/Temporalio/Bridge/src/testing.rs

```diff
@@ -208,6 +209,11 @@ impl TryFrom<&DevServerOptions> for ephemeral_server::TemporalDevServerConfig {
             .port(test_server_options.port())
             .db_filename(options.database_filename.to_option_string())
             .ui(options.ui)
+            .ui_port(if options.ui_port == 0 {
```

Comment:
```suggestion
            .ui_port(if options.ui_port == 0 || !options.ui {
```

I wonder if we need this here